Engine/src/Engine/Scene.h

```diff
@@ -76,11 +77,14 @@ namespace Engine
 
 		//Create quad for sprites
         VerticesComponent createSprite();
+        VerticesComponent createRectangle();
+        void setupVertexAttribPtr(VerticesComponent& vc);
 		
 		//Functions to create sprite specific OpenGL buffers. Returns the ID of said buffers.
-		GLuint getSpriteVBO();
-        GLuint getSpriteVAO();
-        GLuint getSpriteIBO();
+
+		GLuint getVBO(std::string message = "sprite");
```

Comment:
If we're going to use variables to determine what kind of VBO should be generated, let's use an Enum.
It'll be easier to expand and use in the future.